python/cuml/cuml/internals/array.py

```diff
@@ -1172,6 +1172,11 @@ def from_input(
         if (
             not fail_on_order and order != arr.order and order != "K"
         ) or make_copy:
+            info(
+                f"Expected {order} major order but got something else."
```

Comment:
Give the array's order to the user in this warning.
```suggestion
                f"Expected {order} major order but got {arr.order}."
```

Reply:
For the `make_copy` case, I see it's controlled by `make_copy = force_contiguous and not arr.is_contiguous`. Should we issue a different warning message for non-contiguous inputs so the user can tell what went wrong?